pkg/asset/installconfig/platformcredscheck.go

```
@@ -36,6 +36,7 @@ func (a *PlatformCredsCheck) Generate(dependencies asset.Parents) error {
 	switch platform {
 	case aws.Name:
 		_, err = awsconfig.GetSession()
+		err = awsconfig.ValidateAWSCreds()
```

This will swallow any error from the call to `awsconfig.GetSession`.